roborock/roborock_typing.py

```diff
@@ -275,7 +275,7 @@ class CommandInfo:
     RoborockCommand.SET_CUSTOMIZE_CLEAN_MODE: CommandInfo(
         prefix=b"\x00\x00\x00\xa7", params={"data": [], "need_retry": 1}
     ),
-    RoborockCommand.SET_CUSTOM_MODE: CommandInfo(prefix=b"\x00\x00\x00\x87", params=[108]),
+    RoborockCommand.SET_CUSTOM_MODE: CommandInfo(prefix=b"\x00\x00\x00w", params=[108]),
```

Comment:
I had to change this to get it to work - was this change made on purpose?

Reply:
I sniffed commands and merged them into CommandInfoMap. It is strange that the app sends invalid prefixes haha

Reply:
Idk maybe there's some magic resting to prefixes we don't know yet haha. 

Does the prefix appear to be wrong for you as well?

Reply:
Probably. I still don't know much about these 4 heading bytes
Yes, I had to apply your change for it to work.